frontend/src/utils/api.js

```diff
@@ -1,2 +1,73 @@
 //TODO: Add API calls here!
 require("node-fetch");
+const host = "http://localhost:5000/api";
+
+async function getEndpoint(endPoint, dataKey, additonal_headers = null) {
+  try {
+    let headers = { ...additonal_headers, "Content-Type": "application/json" };
+    let response = await fetch(`${host}/${endPoint}`, {
+      method: "GET",
+      headers
+    });
+    let responseJson = await response.json();
+    return dataKey === "" || responseJson.success === false
```

Comment:
?

Reply:
what is dataKey

Reply:
It seems like here you're using `dataKey` to be an extra parameter passed into the function to retrieve data from the returned JSON object. Keeping single responsibility principle in mind, we can delegate that task to the caller.